src/content/docs/id/help/nvidia.md

```
@@ -0,0 +1,111 @@
+---
+title: NVIDIA
+description: Untuk pengguna GPU NVIDIA
+---
+
+Instalasi HyDE mengotomatisasi pemasangan **NVIDIA drivers** dan **nvidia-utils**.
+
+:::note
+Informasi lebih lanjut tersedia di: https://wiki.archlinux.org/title/NVIDIA
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)